### PR TITLE
Operating Model 3.0.1

### PR DESCRIPTION
**Pull request by werner-roman**
### Added

- Data space operations
  - Supply Chain Visibility [BETA]
- How life cycle management
  - Details on patches and participants of gates

### Changed

- Data space operations
  - updated onboarding process & registration
- Data space governance
  - detailing and updating certification of business applications & marketplace
  - updated regulatory framework
- Service Map
  - "Direct collaboration" renamed to "Individual collaboration"
  - detailing the Sandbox Provider [[#88](https://github.com/catenax-eV/catenax-ev.github.io/issues/88)]
- Roles in the Catena-X Ecosystem
  - detailing the Sandbox Provider [[#88](https://github.com/catenax-eV/catenax-ev.github.io/issues/88)]

### CODE REVIEW

File: docs/operating-model/how-data-space-governance/how-data-space-governance.md [hunk 1]
```diff
@@ -115,24 +115,21 @@ Data space participants and their solutions that have a valid certificate past O
...
 
-Companies connected to Catena-X might want to **connect certain internal systems** including company-internal business applications to the Catena-X network. It remains, however, crucial that all participants can rely on a standardized data exchange that guarantees stable data chains built on data sovereignty and interoperability.
+Companies may want to **integrate internal systems** including company-internal business applications to the Catena-X data space. However, tt is crucial that all participants can rely on a standardized data exchange to ensure stable and consistent multi-tier data chains based on data sovereignty and interoperability. Consequently, the following principles apply:
```
Comment (jSchuetz88) [inline docs/operating-model/how-data-space-governance/how-data-space-governance.md]:
```suggestion
Companies may want to **integrate internal systems** including company-internal business applications to the Catena-X data space. However, it is crucial that all participants can rely on a standardized data exchange to ensure stable and consistent multi-tier data chains based on data sovereignty and interoperability. Consequently, the following principles apply:
```

File: docs/operating-model/how-data-space-governance/how-data-space-governance.md [hunk 2]
```diff
@@ -115,24 +115,21 @@ Data space participants and their solutions that have a valid certificate past O
...
 
-Consequently, the following two principles apply:
+- All company-internal business applications required to maintain a stable and consistent multi-tier data (see XXX) must compy with Catena-X standards and be certified by a CAB.
```
Comment (jSchuetz88) [inline docs/operating-model/how-data-space-governance/how-data-space-governance.md]: see xxx?

Comment (jSchuetz88) [reply docs/operating-model/how-data-space-governance/how-data-space-governance.md]: "must compy" -> "must comply"

File: docs/operating-model/how-data-space-governance/how-data-space-governance.md [hunk 3]
```diff
@@ -115,24 +115,21 @@ Data space participants and their solutions that have a valid certificate past O
...
+- All company-internal business applications associated with the Catena-X brand and/or offered on a Catena-X marketplace must comply to Catena-X standards and be certified by a CAB.
 
-- Everything that is necessary to uphold a stable data chain is required to follow Catena-X standards and has to be certified by a CAB.
-- All business applications that are either associated with the Catena-X brand and/or are offered on a Catena-X marketplace are required to follow Catena-X standards and have to be certified by a CAB.
+If an company-internal business applications meets either of these criteria, Catena-X certification is mandatory.
```
Comment (jSchuetz88) [inline docs/operating-model/how-data-space-governance/how-data-space-governance.md]:
```suggestion
If an company-internal business application meets either of these criteria, Catena-X certification is mandatory.
```

Comment (jSchuetz88) [reply docs/operating-model/how-data-space-governance/how-data-space-governance.md]: "meets one of these criteria ..."?

File: docs/operating-model/how-data-space-operations/how-data-space-operations.md [hunk 4]
```diff
@@ -89,10 +93,41 @@ Each data space participant has its own organizational structure, consisting of
...
+
+### Entity-based visibility
+
+In comparison to part-based visibility, entity-based visibility does not differentiate between the individual subcomponents or the assemblies. This means that all subparts and process steps of the respective other legal entity (One Up / N-1) or (One Down / N+1) are provided. In this figure, data is exchanged with Tier-X-1 (1Up) about the part that has a direct relationship with it and the assemblies. It is recommended that supply chain visibility respects symmetrical relationships where granularity of data visibility is equivalent on supplier and customer side.
+
+### BETA
+
+This chapter is released as beta, which therefore is as of now not normative and may change significantly in subsequent iterations. The aim is to provide a definition and a guideline for 1U1D.  In doing so, we want to start the review of the definition - so that we have long-term an overview of which use case has which requirement.
```
Comment (jSchuetz88) [inline docs/operating-model/how-data-space-operations/how-data-space-operations.md]: ggf. konkretisieren, da nicht ganz eindeutig, welches chapter gemeint ist - oder ggf. an den Anfang des Chapters hochziehen?:

> The chapter Supply Chain Visibility (BETA)

**Review by jSchuetz88 — APPROVED** [review]
Only minor findings, thank you!

File: docs/operating-model/how-data-space-governance/how-data-space-governance.md (hunk 2 above)
Comment (werner-roman) [reply docs/operating-model/how-data-space-governance/how-data-space-governance.md]: DONE

File: docs/operating-model/how-data-space-operations/how-data-space-operations.md (hunk 4 above)
Comment (werner-roman) [reply docs/operating-model/how-data-space-operations/how-data-space-operations.md]: Good point - changed

**Pull request merged**